src/Customer/CustomerInf.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Vendor\Service\Customer;
+
+interface CustomerInf
+{
+    public function __construct($cusId);
```

Comment:
I'm not sure about constructor in interface. Move it to abstract class if it used, or simply delete it